tox.ini

```diff
@@ -1,11 +1,12 @@
 [tox]
-envlist = py{27,35,36}-xblock{10,11,12},flake8
+envlist = py{27,35,36,37}-xblock{10,11,12},flake8
```

Comment:
I obviously don't object to adding 3.7 to the test matrix at all, but just curious about the rationale. As far as I can tell the only Ubuntu platform that deploys 3.7 is disco, which is not an LTS and hence, I think, is not a valid deployment platform for Open edX... am I missing something?

Reply:
I could fall back on the future-proofness argument, but the reality is that I put this in because it was the easiest way for me to get a python3 tox run on my Disco box. :grin: 

If you know of a simple way of installing different python versions on bare metal just for testing purposes, please share.  (I know I could just use containers, but I'm looking for a way to integrate test runs into my editor, and that would make it harder.)